src/pages/loginRegister.css

```diff
@@ -0,0 +1,81 @@
+.hero{
+  height: 100%;
+  width: 100%;
+  background: linear-gradient(rgba(0,0,0,0.4),rgba(0,0,0,0.4));
+  background-position: center;
+  background-size: cover;
+  position: absolute;
+}
+.form-box{
+  width: 380px;
+  height: 55%;
+  position: relative;
+  margin: 6% auto;
+  background: #fff;
+  padding: 5px;
+}
+.button-box{
+  width: 220px;
+  margin: 35px auto;
+  position: relative;
+  box-shadow: 0 0 20px 9px #ff61241f;
+  border-radius: 30px;
+}
+.toggle-btn{
+  padding: 10px 30px;
+  cursor: pointer;
+  background: transparent;
+  border: 0;
+  outline: none;
+  position: relative;
+}
+#btn{
```

Comment:
Assign classes to elements to style them, using Ids increases specificity which means if you will try to override styles using a class it won't work.